internal/wsutil/wsutil.go

```diff
@@ -109,48 +81,59 @@ func WebsocketSendStream(conn MessageWriter, r io.Reader, bufferSize int) chan b
 func WebsocketRecvStream(w io.Writer, conn MessageReader) chan bool {
 	ch := make(chan bool)
```

Comment:
Probably this should just be buffered with a size of 1 so the goroutine can complete earlier.

Reply:
True. I changed it to a `close(ch)` instead to match the close in `WebsocketSendStream`.